package.json

```diff
@@ -1253,21 +1252,21 @@
     "http-proxy": "^1.18.1",
     "is-glob": "^4.0.1",
     "is-path-inside": "^3.0.2",
-    "jest": "^26.6.3",
+    "jest": "^27.5.1",
     "jest-axe": "^5.0.0",
-    "jest-canvas-mock": "^2.3.1",
-    "jest-circus": "^26.6.3",
-    "jest-cli": "^26.6.3",
-    "jest-config": "^26",
-    "jest-diff": "^26.6.2",
-    "jest-environment-jsdom": "^26.6.2",
-    "jest-mock": "^26.6.2",
+    "jest-canvas-mock": "^2.4.0",
+    "jest-cli": "^27.5.1",
+    "jest-config": "^27.5.1",
+    "jest-diff": "^27.5.1",
+    "jest-environment-jsdom": "^27.5.1",
+    "jest-matcher-utils": "^27.5.1",
+    "jest-mock": "^27.5.1",
     "jest-raw-loader": "^1.0.1",
-    "jest-runtime": "^26",
+    "jest-runtime": "^27.5.1",
     "jest-silent-reporter": "^0.5.0",
-    "jest-snapshot": "^26.6.2",
-    "jest-specific-snapshot": "^4.0.0",
-    "jest-styled-components": "^7.0.3",
+    "jest-snapshot": "^27.5.1",
+    "jest-specific-snapshot": "^5.0.0",
+    "jest-styled-components": "7.0.3",
```

Comment:
This breaks a bunch of snapshots so I pinned it for now.  There was a fix merged in 7.1.1 but it seems to have made our situation worse.